libraries/AP_ExternalAHRS/AP_ExternalAHRS.h

```diff
@@ -91,7 +89,7 @@ class AP_ExternalAHRS {
     void get_filter_status(nav_filter_status &status) const;
     Vector3f get_gyro(void);
     Vector3f get_accel(void);
-    void send_status_report(mavlink_channel_t chan) const;
+    void send_status_report(class GCS_MAVLINK &link) const;
```

Comment:
sometimes we have "class" in front of the "GCS_MAVLINK &link" argument and other times we don't..

Reply:
Yes.  Where we don't *know* GCS_MAVLINK" is a class because we've included the header, we can tell C++ that it is a class by prepending it with `class`.  Since we're only using a reference here (which is near-enough to a pointer), we don't need to know the storage size of `GCS_MAVLINK` so we can get away without including the header.